app/views/grant_submissions/submissions/reviews/_overall_impact.html.haml

```diff
@@ -16,6 +16,8 @@
             = f.label "#{value}", { class: 'button', for: "overall-#{value}", disabled: disabled }
         %label.scoring-guidance
           Poor
+        -unless disabled
+          = button_tag 'Clear', class: 'secondary button hollow small clear-button', id: "#overall-clear-button"
```

Comment:
Maybe "Clear your selection" or "Clear your score" instead of just "Clear"?  @lmkw may have some thoughts.

Also, maybe this should go below the button group rather than next to it? Again, Laura may have some thoughts.

Reply:
Talked with @lmkw, kept the wording and moved the button slightly to the right.